oaknational/Oak-Web-Application
# Added new curriculum download UI section

## Description
Added download UI section for curriculum download. Also added "Back to KS4 options" button into the UI.


## Issue(s)
Fixes `CUR-778`


## How to test

1. Go to {owa_deployment_url}/teachers/curriculum/science-secondary-edexcel/downloads
2. Check for "Back to KS4 options" button
3. Check UI implemented as expected

## Screenshots

<img width="363" alt="Screenshot 2024-08-08 at 16 25 04" src="https://github.com/user-attachments/assets/da5259d9-c51d-4a10-8cce-ecb8a8ee11df"> <img width="363" alt="Screenshot 2024-08-08 at 16 25 10" src="https://github.com/user-attachments/assets/591a6a99-8f87-418b-b84f-37a2c2364566">
